app/views/CancelVATRegistrationPage.scala.html

```diff
@@ -27,12 +28,15 @@
     formHasErrors = form.errors.nonEmpty,
     formErrors = form.errors,
     optPageMode = Some(pageMode)) {
+    @pageHeader(messages("cancelVATRegistration.headingAndTitle"), showPageHeader = false)
```

Comment:
Maybe `showPageHeader` isn't needed? Could we change the first parameter of the page header to be Option[String]?